From 7fa9e9b683f1c573c58a14755347988919bc7d06 Mon Sep 17 00:00:00 2001
From: YenLin Pan <yenlin.pan@zyxel.com.tw>
Date: Wed, 14 May 2025 13:47:06 +0800
Subject: [PATCH] pinctrl: make the switch LED works

Enable switch LED pin definition for LED0/LED1/LED2 control

Signed-off-by: YenLin Pan <YenLin.Pan@zyxel.com.tw>
---
 src/init/ssdk_mht_pinctrl.c | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/src/init/ssdk_mht_pinctrl.c b/src/init/ssdk_mht_pinctrl.c
index 2debe59..1ae0002 100755
--- a/src/init/ssdk_mht_pinctrl.c
+++ b/src/init/ssdk_mht_pinctrl.c
@@ -33,11 +33,17 @@ static struct mht_pinctrl_setting mht_pin_settings[] = {
     /*PINs default MUX Setting*/
     MHT_PIN_SETTING_MUX(0,  MHT_PIN_FUNC_INTN_WOL),
     MHT_PIN_SETTING_MUX(1,  MHT_PIN_FUNC_INTN),
-#if 0
+#if 1
     MHT_PIN_SETTING_MUX(2,  MHT_PIN_FUNC_P0_LED_0),
     MHT_PIN_SETTING_MUX(3,  MHT_PIN_FUNC_P1_LED_0),
     MHT_PIN_SETTING_MUX(4,  MHT_PIN_FUNC_P2_LED_0),
     MHT_PIN_SETTING_MUX(5,  MHT_PIN_FUNC_P3_LED_0),
+    MHT_PIN_SETTING_MUX(6,  MHT_PIN_FUNC_P0_LED_2),
+    MHT_PIN_SETTING_MUX(7,  MHT_PIN_FUNC_P1_LED_2),
+    MHT_PIN_SETTING_MUX(8,  MHT_PIN_FUNC_P2_LED_2),
+    MHT_PIN_SETTING_MUX(9,  MHT_PIN_FUNC_P3_LED_2),
+#endif
+#if 0
     MHT_PIN_SETTING_MUX(6,  MHT_PIN_FUNC_PPS_IN),
     MHT_PIN_SETTING_MUX(7,  MHT_PIN_FUNC_TOD_IN),
     MHT_PIN_SETTING_MUX(8,  MHT_PIN_FUNC_RTC_REFCLK_IN),
@@ -49,7 +55,7 @@ static struct mht_pinctrl_setting mht_pin_settings[] = {
     MHT_PIN_SETTING_MUX(13, MHT_PIN_FUNC_P0_TOD_OUT),
     MHT_PIN_SETTING_MUX(14, MHT_PIN_FUNC_P0_CLK125_TDI),
     MHT_PIN_SETTING_MUX(15, MHT_PIN_FUNC_P0_SYNC_CLKO_PTP),
-#if 0
+#if 1
     MHT_PIN_SETTING_MUX(16, MHT_PIN_FUNC_P0_LED_1),
     MHT_PIN_SETTING_MUX(17, MHT_PIN_FUNC_P1_LED_1),
     MHT_PIN_SETTING_MUX(18, MHT_PIN_FUNC_P2_LED_1),
-- 
2.34.1

